Sources/UXKit/Common/UXViewController.swift

```diff
@@ -79,4 +79,14 @@
   public var  uxChildren : [ UXViewController ]  { return children }
   public func uxRemoveFromParent()               { removeFromParent() }
   public var  uxParent   : UXViewController?     { return parent }
+    
+#if os(macOS)
+    public func dismiss(animated flag: Bool, completion: (() -> Void)? = nil) {
+        self.dismiss(self)
+    }
+
+    public func present(_ viewControllerToPresent: UXViewController, animated flag: Bool, completion: (() -> Void)? = nil) {
+        self.presentAsModalWindow(viewControllerToPresent)
```

Comment:
Maybe that should also have `modalPresentationStyle` then (e.g. this also seems to support popovers in UIKit). Also the safer default variant seems to be presenting as sheets (because they can nest)?